From f57a6637c94e07fd0af7e3ae5c3fdb842bccdc3c Mon Sep 17 00:00:00 2001
From: Mario Limonciello <mario.limonciello@amd.com>
Date: Tue, 4 Jan 2022 14:16:39 -0600
Subject: [PATCH] trivial: fix man page sections

They're marked in the man page as section 8 but installed into 1.
They should be installed into 1 as they're not admin tools.
---
 plugins/dfu/dfu-tool.1          | 2 +-
 plugins/uefi-capsule/fwupdate.1 | 2 +-
 plugins/uefi-dbx/dbxtool.1      | 2 +-
 src/fwupdagent.1                | 2 +-
 src/fwupdmgr.1                  | 2 +-
 src/fwupdtool.1                 | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/plugins/dfu/dfu-tool.1 b/plugins/dfu/dfu-tool.1
index 3e10fa1f..831d2e14 100644
--- a/plugins/dfu/dfu-tool.1
+++ b/plugins/dfu/dfu-tool.1
@@ -1,5 +1,5 @@
 .\" Report problems in https://github.com/fwupd/fwupd
-.TH man 8 "11 April 2021" @PACKAGE_VERSION@ "dfu-tool man page"
+.TH man 1 "11 April 2021" @PACKAGE_VERSION@ "dfu-tool man page"
 .SH NAME
 dfu-tool \- write firmware to DFU devices
 .SH SYNOPSIS
diff --git a/plugins/uefi-capsule/fwupdate.1 b/plugins/uefi-capsule/fwupdate.1
index 87fbde1d..c41e55e0 100644
--- a/plugins/uefi-capsule/fwupdate.1
+++ b/plugins/uefi-capsule/fwupdate.1
@@ -1,5 +1,5 @@
 .\" Report problems in https://github.com/fwupd/fwupd
-.TH man 8 "11 April 2021" @PACKAGE_VERSION@ "fwupdate man page"
+.TH man 1 "11 April 2021" @PACKAGE_VERSION@ "fwupdate man page"
 .SH NAME
 fwupdate \-ddebugging utility for UEFI firmware updates
 .SH SYNOPSIS
diff --git a/plugins/uefi-dbx/dbxtool.1 b/plugins/uefi-dbx/dbxtool.1
index 6b795ef6..ea4453b5 100644
--- a/plugins/uefi-dbx/dbxtool.1
+++ b/plugins/uefi-dbx/dbxtool.1
@@ -1,5 +1,5 @@
 .\" Report problems in https://github.com/fwupd/fwupd
-.TH man 8 "11 April 2021" @PACKAGE_VERSION@ "dbxtool man page"
+.TH man 1 "11 April 2021" @PACKAGE_VERSION@ "dbxtool man page"
 .SH NAME
 dbxtool \- modify the dbx revokation list
 .SH SYNOPSIS
diff --git a/src/fwupdagent.1 b/src/fwupdagent.1
index 6beea4ee..23406d59 100644
--- a/src/fwupdagent.1
+++ b/src/fwupdagent.1
@@ -1,5 +1,5 @@
 .\" Report problems in https://github.com/fwupd/fwupd
-.TH man 8 "11 April 2021" @PACKAGE_VERSION@ "fwupdagent man page"
+.TH man 1 "11 April 2021" @PACKAGE_VERSION@ "fwupdagent man page"
 .SH NAME
 fwupdagent \- firmware updating agent
 .SH SYNOPSIS
diff --git a/src/fwupdmgr.1 b/src/fwupdmgr.1
index c9bbc013..17a06807 100644
--- a/src/fwupdmgr.1
+++ b/src/fwupdmgr.1
@@ -1,5 +1,5 @@
 .\" Report problems in https://github.com/fwupd/fwupd
-.TH man 8 "11 April 2021" @PACKAGE_VERSION@ "fwupdmgr man page"
+.TH man 1 "11 April 2021" @PACKAGE_VERSION@ "fwupdmgr man page"
 .SH NAME
 fwupdmgr \- firmware update manager client utility
 .SH SYNOPSIS
diff --git a/src/fwupdtool.1 b/src/fwupdtool.1
index 0dbd452c..ae53fb4b 100644
--- a/src/fwupdtool.1
+++ b/src/fwupdtool.1
@@ -1,5 +1,5 @@
 .\" Report problems in https://github.com/fwupd/fwupd
-.TH man 8 "11 April 2021" @PACKAGE_VERSION@ "fwupdtool man page"
+.TH man 1 "11 April 2021" @PACKAGE_VERSION@ "fwupdtool man page"
 .SH NAME
 fwupdtool \- standalone firmware update utility
 .SH SYNOPSIS
-- 
2.25.1

